pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -550,11 +562,13 @@ func processEvents(image string, logChan <-chan logEntry,
 				return
 			}
 			if event.isAppLog {
-				appUUID = lookupDomainName(ctx, event.source)
-				if appUUID == "" {
-					log.Errorf("processEvents(%s): UUID for App instance %s not found",
-						image, event.source)
-					break
+				if event.appUUID == "" {
```

Comment:
What would happen to VM logs?

Reply:
@gkodali-zededa VM logs still sent the same way. Here it does a domain-name lookup to find the UUID and format the URL endpoint. For container logs, the App name is not domain-name, and we pass this UUID along, so since they both are 'applog', one needs to do the lookup here.

Reply:
it's something like this output from webpage (to have both types of logs):

{"container":"edgeAgent","log-time":"2020-06-27T23:52:36.360044183Z","message":"Done sending 500 messages"}
{"container":"edgeAgent","log-time":"2020-06-28T18:40:26.352104742Z","message":" [INF] - Entering periodic task to reauthenticate connected clients"}
{"source":"guest_vm-azure-iot.1", "time":"2020-06-28T18:43:31.774876+00:00", "level":"emerg", "msg":"2020-06-28T18:43:31Z,guest_vm-azure-iot.1;40244fe4-d1e4-4591-b201-f2302d78ab32 login: ", "pid":0, "partition":"IMGA"}

Reply:
thanks @naiming-zededa